From ad0af631250ce3399127d42995cc0bef6ad3ee5e Mon Sep 17 00:00:00 2001
From: Herman Rimm <herman@rimm.ee>
Date: Sun, 18 Aug 2024 16:05:44 +0200
Subject: [PATCH] libguile-ssh/session-func: Fix variable name

* libguile-ssh/session-func.c (set_bool_opt): Rename bool to boolean.
---
 libguile-ssh/session-func.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/libguile-ssh/session-func.c b/libguile-ssh/session-func.c
index 7006b62..b379472 100644
--- a/libguile-ssh/session-func.c
+++ b/libguile-ssh/session-func.c
@@ -228,18 +228,18 @@ set_int32_opt (ssh_session session, int type, SCM value)
   return ssh_options_set (session, type, &c_value);
 }
 
-/* Convert VALUE to integer that represents a boolan value (0
+/* Convert VALUE to integer that represents a boolean value (0
    considered as false, any other value is true), and pass it to
    ssh_options_set */
 static inline int
 set_bool_opt (ssh_session session, int type, SCM value)
 {
-  int32_t bool;
+  int32_t boolean;
 
   SCM_ASSERT (scm_is_bool (value), value, SCM_ARG3, "session-set!");
 
-  bool = scm_to_bool (value);
-  return ssh_options_set (session, type, &bool);
+  boolean = scm_to_bool (value);
+  return ssh_options_set (session, type, &boolean);
 }
 
 /* Convert VALUE to a socket file descriptor and pass it to
-- 
2.45.2

